expected-indents/src/static/script_game.js

```diff
@@ -0,0 +1,152 @@
+let lifesign = "❤️";
+let uid = document.getElementById("uid").innerText;
+// console.log(`uid: ${uid}`);
+let liveselement = document.getElementById("lives");
+let imageelement = document.getElementById("hint");
+let wordtable = document.getElementById("wordtable");
+let warning = document.getElementById("warning");
+let modal = document.getElementById("winorlose");
+let modal_text = document.getElementById("winorlose-text");
+let feedback_text = document.getElementById("feedback");
+let guessedletters = [];
+let lives = 0;
+let cache_busting = 0;
+
+
+function guess_character(guessedLetter) {
+	console.log("You entered: " + guessedLetter);
+
+	// Check if the input is valid
+	check = check_input(guessedLetter);
+
+	if (check) {
+		guessedletters.push(guessedLetter);  // Add the entered letter to the guessedletters array
+		console.log(guessedletters);
+		fetch(`/guess/${uid}/${guessedLetter}`) // Send a GET request to the API endpoint
+			.then(response => response.json())
+			.then(data => {
+				/*
+				I forgot exactly how we decided the API response should be... so I assume it's like this:
+				{
+					"word": "w_r_",
+					"lives": 5,
+					"image": "https://example.com/image.png"
+				}
+				*/
```

Comment:
This comment is a bit worrying, both it existing in first place as well as it still being there in the PR